src/Microsoft.PowerShell.SHiPS/SHiPSDrive.cs

```diff
@@ -273,6 +274,16 @@ private static Match TryExcludeFQProviderPath(string rootPath)
             var match = ModuleAndTypeRegex.Match(leaf);
             return match.Success ? match : null;
         }
+
+        internal static string HomePath
+        {
+            get
+            {
+                var path =  RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "USERPROFILE" : "HOME";
```

Comment:
`$HOME` is universal for PowerShell isn't it? Is there a case for Windows where it won't be set?

Reply:
UserProfile is cmd shell / OS specific. I believe HOME is always available in PowerShell

Reply:
Yes home is variable exists in PowerShell. I wanted to avoid calling PowerShell cmdlet from C# code because it is slow.  However comparing to upgrading .net version which will require people to install the latest .net,  calling PowerShell Get-Variable possibly wins.

Reply:
Good point. This is C#, so we should use the C# APIs. Ignore my original request. The existing code is fine as long as it is compliant with .Net Standard 2.0.